Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -74,6 +78,11 @@ class HomeFragmentViewModel @AssistedInject constructor(
 
     private var isLoweredRiskLevelDialogBeingShown = false
 
+    suspend fun observeTestResultToSchedulePositiveTestResultReminder() =
+        submissionCardsStateProvider.state
+            .first { it.isPositiveSubmissionCardVisible() }
```

Comment:
After an app reset via the settings menu, the deviceUiState is still "PAIRED_POSITIVE" which will falsely trigger the notification. This needs to be fixed.

Reply:
Resolved. The SubmissionRepository state has not been reset on app reset.